backend/uclapi/workspaces/occupeye/archive.py

```diff
@@ -0,0 +1,206 @@
+from datetime import datetime, timedelta, timezone
+from typing import List
+
+from workspaces.models import Surveys, Historical, Sensors, SensorReplacements, SurveyChanges
+from .constants import OccupEyeConstants
+from .endpoint import OccupeyeEndpoint, Endpoint
+
+FIRST_OCCUPEYE_INSTALLATION = datetime.strptime("2017-01-01T00:00:00+0000", "%Y-%m-%dT%H:%M:%S%z")
+MAX_TIME_DELTA = timedelta(days=28)
+
+
+class OccupEyeArchive:
+    def __init__(self, endpoint: Endpoint = None):
+        if endpoint is None:
+            endpoint = OccupeyeEndpoint()
```

Comment:
It is bad practice to have mutable default parameter values in Python. There is a good explanation [here](https://web.archive.org/web/20200221224620/http://effbot.org/zone/default-values.htm). I believe it is technically safe in this case because we are only ever creating one archive when executing a command but if we ever create another archive we may get unexpected behaviour.